### PR TITLE
Add installation instructions for FreeBSD

### PR DESCRIPTION
**Pull request by whitequark**

### CODE REVIEW

File: docs/manual/src/install.rst [hunk 1]
```diff
@@ -138,12 +138,52 @@ If these instructions don't work for you, please `file it as a bug <file-a-bug_>
         $ glasgow --version
         $ glasgow build --rev C3 uart
 
+.. tab:: FreeBSD
+
+    You will need to have `pipx`_, `Yosys`_, `nextpnr`_, and `icestorm`_ installed. To install these packages, run:
+
+    .. code-block:: console
+
+        $ sudo pkg install pipx yosys nextpnr icestorm
```
Comment (emaste) [inline docs/manual/src/install.rst:147]: fwiw our pipx package doesn't depend on pip, but it seems requires it for pipx uninstall

File: docs/manual/src/install.rst [hunk 2]
```diff
@@ -138,12 +138,52 @@ If these instructions don't work for you, please `file it as a bug <file-a-bug_>
         $ glasgow --version
         $ glasgow build --rev C3 uart
 
+.. tab:: FreeBSD
+
+    You will need to have `pipx`_, `Yosys`_, `nextpnr`_, and `icestorm`_ installed. To install these packages, run:
```
Comment (emaste) [inline docs/manual/src/install.rst:143]: also need `abc`

for the toolchain I did:
`pkg install abc yosys nextpnr icestorm`

File: docs/manual/src/install.rst (hunk 1 above)
Comment (whitequark) [reply docs/manual/src/install.rst]: fixed

File: docs/manual/src/install.rst (hunk 2 above)
Comment (whitequark) [reply docs/manual/src/install.rst:143]: fixed

**Review by emaste — APPROVED** [review]

**Pull request merged**